egs/librispeech/s5/local/prepare_dict.sh

```diff
@@ -72,10 +72,15 @@ if [ $stage -le 1 ]; then
   auto_lexicon_prefix="$g2p_dir/lexicon_autogen"
 
   mkdir -p $g2p_dir/log
-  auto_vocab_splits=$(eval "echo $auto_vocab_prefix.{$(seq -s',' $nj | sed 's/,$//')}")
-  awk 'NR==FNR{a[$1] = 1; next} !($1 in a)' $cmudict_plain $vocab |\
-    sort | tee $g2p_dir/vocab_autogen.full |\
-    utils/split_scp.pl /dev/stdin $auto_vocab_splits || exit 1
+  if [ ! $nj -eq 1 ];
```

Comment:
also, please could you write it in this way? it's more compact and more readable, I believe
``` 
if [ $nj -eq 1 ]; then
  auto_vocab_splits="${auto_vocab_prefix}.1"  #If nj is 1, use vocab_autogen.1 instead of .{1}
else
  auto_vocab_splits=$(eval "echo $auto_vocab_prefix.{$(seq -s',' $nj | sed 's/,$//')}") #Create the list of files to split
fi
```

Reply:
Ok, looking at it

Reply:
Thanks for this update. This indeed is more readable, fixed it.